tailwind.config.js

```diff
@@ -1,25 +1,62 @@
+import { fontFamily } from 'tailwindcss/defaultTheme';
+
 /** @type {import('tailwindcss').Config} */
-export default {
-	mode: 'jit',
-	darkMode: 'class',
-	content: ['./src/**/*.{html,js,svelte,ts}', './node_modules/flowbite-svelte/**/*.{html,js,svelte,ts}'],
```

Comment:
I think you need to keep the Flowbite content here (and remove Flowbite in a different PR)

Reply:
ahh yeah, I didn't realize that stuff got removed.